usefulUtilities/sitV2/appResources/appData/sitClientV2.js

```diff
@@ -113,6 +113,10 @@
 
         // Set isSitting value in Settings
         var sitNewSettings = [_this.entityID];
+        changedSeats = false;
+        if(Settings.getValue(SETTING_KEY_AVATAR_SITTING, false) !== false && Settings.getValue(SETTING_KEY_AVATAR_SITTING, false)[0] !== _this.entityID){
```

Comment:
```suggestion
        if (Settings.getValue(SETTING_KEY_AVATAR_SITTING, false) !== false && Settings.getValue(SETTING_KEY_AVATAR_SITTING, false)[0] !== _this.entityID){
```

Reply:
Ack

Reply:
please fix - looks like you missed this in your latest commit (or didn't do it)